src/openvic-simulation/types/Date.hpp

```diff
@@ -109,7 +109,9 @@ namespace OpenVic {
 		static constexpr Timespan::day_t MONTHS_IN_YEAR = DAYS_IN_MONTH.size();
 		static constexpr Timespan::day_t MAX_DAYS_IN_MONTH = *ranges::max_element(DAYS_IN_MONTH);
 
-		static constexpr Timespan::day_t DAYS_IN_YEAR = std::accumulate(DAYS_IN_MONTH.begin(), DAYS_IN_MONTH.end(), 0);
+		static constexpr Timespan::day_t DAYS_IN_YEAR = std::accumulate(
+			DAYS_IN_MONTH.begin(), DAYS_IN_MONTH.end(), static_cast<Timespan::day_t>(0)
```

Comment:
Pretty sure you can do
```suggestion
			DAYS_IN_MONTH.begin(), DAYS_IN_MONTH.end(), Timespan::day_t { 0 }
```

Reply:
sure